.travis.yml

```diff
@@ -49,7 +49,7 @@ install:
     - PHPUNIT="$(readlink -f ./vendor/bin/phpunit) --colors=always"
 
 script:
-    - if [ "$deps" = "no" ]; then echo "$components" | parallel --gnu --keep-order 'echo -e "\\nRunning {} tests"; '$PHPUNIT' --exclude-group tty,benchmark,intl-data {} || (echo -e "\\e[41mKO\\e[0m {}" && $(exit 1));'; fi;
+    - if [ "$deps" = "no" ]; then echo "$components" | parallel --gnu --keep-order 'echo -e "\\nRunning {} tests"; '"$PHPUNIT"' --exclude-group tty,benchmark,intl-data {} || (echo -e "\\e[41mKO\\e[0m {}" && $(exit 1));'; fi;
```

Comment:
shoudln't you remove the single quotes rather than just adding double quotes inside them ?